.pre-commit-config.yaml

```diff
@@ -41,3 +41,11 @@ repos:
         stages: [commit]
         types: [python]
         require_serial: true
+
+      - id: isort
```

Comment:
Looks like most other STAC repos do not use local hooks. Is there a reason why pystac uses them?

Reply:
Not that I know of -- there wasn't a lot of justification in the original issue: https://github.com/stac-utils/pystac/issues/390#issue-908961530.